README.md

```diff
@@ -1,6 +1,6 @@
 # git-herald
 This serverless application will take github events
-(initially new pull requests) and end them to your
+(initially new pull requests) and send them to your
```

Comment:
suggest change to `forward`... "and forward them to your"

Reply:
I like this change